# Catalan translation for update-manager
# Copyright (C) 2006
# This file is distributed under the same license as the update-manager package.
# Jordi Irazuzta Cardús <irazuzta@gmail.com>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2015-10-07 09:44-0700\n"
"PO-Revision-Date: 2015-03-20 12:20+0000\n"
"Last-Translator: Walter Garcia-Fontes <walter.garcia@upf.edu>\n"
"Language-Team: Catalan <tradgnome@softcatala.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2018-04-25 05:36+0000\n"
"X-Generator: Launchpad (build 18623)\n"
"Language: ca\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Servidor per %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Servidor principal"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Servidors personalitzats"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "No s'ha pogut calcular l'entrada del fitxer sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"No s'ha pogut trobar cap paquet, esteu utilitzat un disc de l'Ubuntu per a "
"l'arquitectura adequada?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "No s'ha pogut afegir el CD"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"S'ha produït un error en afegir el CD i l'actualització s'ha cancel·lat. "
"Informeu d'aquest error si això ha passat amb un CD d'Ubuntu vàlid.\n"
"\n"
"El missatge d'error ha estat:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeCache.py:180
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Elimina el paquet en mal estat"
msgstr[1] "Elimina els paquets en mal estat"

#: ../DistUpgrade/DistUpgradeCache.py:183
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"El paquet «%s» es troba en un estat inconsistent i s'ha de tornar a "
"instal·lar, però no s'ha trobat l'arxiu per a fer-ho. Voleu eliminar aquest "
"paquet i continuar?"
msgstr[1] ""
"Els paquets «%s» es troben en un estat inconsistent i s'han de tornar a "
"instal·lar, però no s'ha trobat els arxius per a fer-ho. Voleu eliminar "
"aquests paquets i continuar?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:285
msgid "The server may be overloaded"
msgstr "Pot ser que el servidor estigui sobrecarregat"

#: ../DistUpgrade/DistUpgradeCache.py:398
msgid "Broken packages"
msgstr "Paquets trencats"

#: ../DistUpgrade/DistUpgradeCache.py:399
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"El vostre sistema conté paquets trencats que no es poden arreglar amb "
"aquesta aplicació. Utilitzeu el Synaptic o l'apt-get abans de continuar."

#. FIXME: change the text to something more useful
#: ../DistUpgrade/DistUpgradeCache.py:670
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" This can be caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
" * Running the current pre-release version of Ubuntu\n"
" * Unofficial software packages not provided by Ubuntu\n"
"\n"
msgstr ""
"S'ha produït un problema irresoluble mentre es feia el càlcul per a "
"l'actualització.\n"
"\n"
" Això pot ser degut a:\n"
" * l'actualització a una versió en desenvolupament de l'Ubuntu\n"
" * l'execució de l'actual versió en desenvolupament de l'Ubuntu\n"
" * la instal·lació de paquets no oficials de programari no proveïts per "
"l'Ubuntu\n"
"\n"

#: ../DistUpgrade/DistUpgradeCache.py:680
msgid "This is most likely a transient problem, please try again later."
msgstr ""
"Això és probablement un problema transitori, torneu a provar-ho més tard."

#: ../DistUpgrade/DistUpgradeCache.py:683
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Si res d'això és aplicable, informeu d'aquest error utilitzant l'ordre "
"«ubuntu-bug ubuntu-release-upgrader-core» en un terminal."

#: ../DistUpgrade/DistUpgradeCache.py:688
msgid "Could not calculate the upgrade"
msgstr "No s'ha pogut calcular l'actualització"

#: ../DistUpgrade/DistUpgradeCache.py:747
msgid "Error authenticating some packages"
msgstr "S'ha produït un error en autenticar alguns paquets"

#: ../DistUpgrade/DistUpgradeCache.py:748
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"No s'han pogut autenticar alguns paquets. Potser hi ha un problema temporal "
"amb la xarxa. Podeu provar-ho més tard. A continuació es mostra la llista "
"amb els paquets no autenticats."

#: ../DistUpgrade/DistUpgradeCache.py:769
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr ""
"El paquet «%s» està marcat per a eliminar-lo, però apareix a la llista negra "
"de fitxers a eliminar."

#: ../DistUpgrade/DistUpgradeCache.py:775
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "El paquet essencial «%s» està marcat per a ésser eliminat."

#: ../DistUpgrade/DistUpgradeCache.py:784
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "S'està intentant instal·lar la versió «%s» de la llista negra"

#: ../DistUpgrade/DistUpgradeCache.py:903
#, python-format
msgid "Can't install '%s'"
msgstr "No s'ha pogut instal·lar «%s»"

#: ../DistUpgrade/DistUpgradeCache.py:904
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""
"Ha sigut impossible instal·lar el paquet necessari. Informeu d'aquest error "
"utilitzant l'ordre «ubuntu-bug ubuntu-release-upgrader-core» en un terminal."

#: ../DistUpgrade/DistUpgradeCache.py:915
msgid "Can't guess meta-package"
msgstr "No s'ha pogut conjecturar el metapaquet"

#: ../DistUpgrade/DistUpgradeCache.py:916
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""
"El vostre sistema no conté un paquet %s o %s i no ha estat possible detectar "
"la versió de l'Ubuntu que esteu fent servir.\n"
" Instal·leu un dels paquets mencionats fent servir el synaptic o l'apt-get "
"abans de continuar."

#: ../DistUpgrade/DistUpgradeController.py:118
msgid "Reading cache"
msgstr "S'està llegint la memòria cau"

#: ../DistUpgrade/DistUpgradeController.py:222
msgid "Unable to get exclusive lock"
msgstr "No s'ha pogut obtenir un bloqueig exclusiu"

#: ../DistUpgrade/DistUpgradeController.py:223
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Normalment això es deu al fet que teniu un altre gestor de paquets en "
"execució (com  ara l'apt-get o l'aptitude) i cal que abans el tanqueu."

#: ../DistUpgrade/DistUpgradeController.py:281
msgid "Upgrading over remote connection not supported"
msgstr "No és possible actualitzar a través d'una connexió remota"

#: ../DistUpgrade/DistUpgradeController.py:282
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Esteu duent a terme l'actualització a través d'una connexió SSH remota amb "
"un frontal que no admet aquesta funció. Intenteu realitzar una actualització "
"en mode text amb l'ordre «do-release-upgrade». \n"
"\n"
"S'interromprà l'actualització. Torneu-ho a intentar sense SSH."

#: ../DistUpgrade/DistUpgradeController.py:296
msgid "Continue running under SSH?"
msgstr "Voleu continuar treballant via SSH?"

#: ../DistUpgrade/DistUpgradeController.py:297
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Segons sembla, aquesta sessió s'està executant per sota de SSH. Actualment "
"no és recomanable realitzar una actualització per sota de SSH, atès que en "
"cas de fallada la recuperació és més difícil.\n"
"\n"
"Si continueu, s'iniciarà un dimoni addicional al port «%s».\n"
"Voleu continuar?"

#: ../DistUpgrade/DistUpgradeController.py:311
msgid "Starting additional sshd"
msgstr "S'està iniciant un sshd addicional"

#: ../DistUpgrade/DistUpgradeController.py:312
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"Per facilitar la recuperació en cas de fallada, s'iniciarà un sshd "
"addicional al port «%s». Si alguna cosa anés malament amb l'ssh en ús, podeu "
"fer servir l'addicional.\n"

#: ../DistUpgrade/DistUpgradeController.py:320
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Si feu servir un tallafoc, necessitareu obrir temporalment aquest port. Atès "
"que això és potencialment perillós, no es fa automàticament. Per exemple, "
"podeu obrir el port amb:\n"
"«%s»"

#: ../DistUpgrade/DistUpgradeController.py:393
#: ../DistUpgrade/DistUpgradeController.py:417
msgid "Can not upgrade"
msgstr "No es pot actualitzar"

#: ../DistUpgrade/DistUpgradeController.py:394
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Aquesta eina no permet l'actualització de «%s» a «%s»."

#: ../DistUpgrade/DistUpgradeController.py:407
msgid "Sandbox setup failed"
msgstr "Ha fallat la configuració d'un entorn de proves"

#: ../DistUpgrade/DistUpgradeController.py:408
msgid "It was not possible to create the sandbox environment."
msgstr "No ha estat possible crear un entorn de proves."

#: ../DistUpgrade/DistUpgradeController.py:414
msgid "Sandbox mode"
msgstr "Mode de prova"

#: ../DistUpgrade/DistUpgradeController.py:415
#, python-format
msgid ""
"This upgrade is running in sandbox (test) mode. All changes are written to "
"'%s' and will be lost on the next reboot.\n"
"\n"
"*No* changes written to a system directory from now until the next reboot "
"are permanent."
msgstr ""
"Aquesta actualització s'està executant en mode de proves. Tots els canvis "
"s'escriuran a «%s» i es perdran en reiniciar.\n"
"\n"
"Cap dels canvis fets als directoris del sistema seran permanents."

#: ../DistUpgrade/DistUpgradeController.py:418
msgid ""
"Your python install is corrupted. Please fix the '/usr/bin/python' symlink."
msgstr ""
"La vostra instal·lació del Python està malmesa. Reviseu l'enllaç simbòlic "
"«/usr/bin/python»."

#: ../DistUpgrade/DistUpgradeController.py:447
#, python-format
msgid "Can not write to '%s'"
msgstr "No es pot escriure a «%s»"

#: ../DistUpgrade/DistUpgradeController.py:448
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""
"No es pot escriure al directori del sistema «%s», per la qual cosa "
"l'actualització no pot continuar.\n"
"Assegureu-vos que aquest directori tingui els permisos d'escriptura adients."

#: ../DistUpgrade/DistUpgradeController.py:459
msgid "Include latest updates from the Internet?"
msgstr "Voleu incloure les darreres actualitzacions d'Internet?"

#: ../DistUpgrade/DistUpgradeController.py:460
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"El sistema d'actualitzacions pot utilitzar Internet per baixar i instal·lar "
"automàticament les últimes actualitzacions. Si disposeu d'una connexió de "
"xarxa, aquesta opció és molt recomanable.\n"
"\n"
"L'actualització trigarà més, però en acabar tindreu un sistema completament "
"actualitzat. Podeu optar per no fer-ho, però aviat haureu d'actualitzar el "
"sistema.\n"
"Si responeu «no» la xarxa no s'utilitzarà per a res."

#: ../DistUpgrade/DistUpgradeController.py:634
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""
"No està pensat per a éssers humans mentre %s està en fase de desenvolupament"

#: ../DistUpgrade/DistUpgradeController.py:719
#, python-format
msgid "disabled on upgrade to %s"
msgstr "desactivat en actualitzar a %s"

#: ../DistUpgrade/DistUpgradeController.py:789
msgid "No valid mirror found"
msgstr "No s'ha trobat cap rèplica vàlida"

#: ../DistUpgrade/DistUpgradeController.py:790
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""
"No s'han trobat dipòsits de programari per a l'actualització. Això pot "
"passar quan s'utilitza una rèplica interna o quan la informació de la "
"rèplica no es troba actualitzada.\n"
"\n"
"Voleu reescriure 'sources.list' igualment? Si seleccioneu 'Sí' "
"s'actualitzaran totes les entrades '%s' per '%s'.\n"
"Si seleccioneu 'No' l'actualització es cancel·larà."

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:810
msgid "Generate default sources?"
msgstr "Voleu generar les fonts per defecte?"

#: ../DistUpgrade/DistUpgradeController.py:811
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"No s'ha trobat cap entrada vàlida per a «%s» en analitzar el fitxer "
"«sources.list».\n"
"\n"
"Voleu que s'afegeixin entrades per a «%s»? Si trieu que «No», es cancel·larà "
"l'actualització."

#: ../DistUpgrade/DistUpgradeController.py:846
#: ../DistUpgrade/DistUpgradeController.py:852
msgid "Repository information invalid"
msgstr "La informació dels dipòsits no és vàlida"

#: ../DistUpgrade/DistUpgradeController.py:847
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""
"En actualitzar la informació dels dipòsits s'ha obtingut un fitxer no vàlid, "
"per la qual cosa s'iniciarà el procés d'informe d'errors."

#: ../DistUpgrade/DistUpgradeController.py:853
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"L'actualització de la informació del dipòsit ha finalitzat amb un fitxer "
"corrupte. Per informar un error instal·leu apport i executeu  'apport-bug "
"ubuntu-release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:862
msgid "Third party sources disabled"
msgstr "S'han desactivat les fonts de tercers"

#: ../DistUpgrade/DistUpgradeController.py:863
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"S'han desactivat algunes entrades de tercers. Les podeu reactivar després de "
"l'actualització des de l'eina «software-properties» en el vostre gestor de "
"paquets."

#: ../DistUpgrade/DistUpgradeController.py:904
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Paquet en estat inconsistent"
msgstr[1] "Paquets en estat inconsistent"

#: ../DistUpgrade/DistUpgradeController.py:907
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"El paquet «%s» es troba en un estat inconsistent i s'ha de tornar a "
"instal·lar, però no s'ha trobat l'arxiu per a fer-ho. Torneu-lo a instal·lar "
"manualment o suprimiu-lo del sistema."
msgstr[1] ""
"Els paquets «%s» es troben en un estat inconsistent i s'han de tornar a "
"instal·lar, però no s'ha trobat els arxius per a fer-ho. Torneu-los a "
"instal·lar manualment o suprimiu-los del sistema."

#: ../DistUpgrade/DistUpgradeController.py:964
msgid "Error during update"
msgstr "S'ha produït un error en l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:965
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"S'ha produït un error mentre s'actualitzava el vostre sistema. Normalment "
"això és a causa de problemes de xarxa. Comproveu la vostra connexió de xarxa "
"i torneu a intentar-ho."

#: ../DistUpgrade/DistUpgradeController.py:1021
msgid "Not enough free disk space"
msgstr "No disposeu de suficient espai al disc"

#: ../DistUpgrade/DistUpgradeController.py:942
#, python-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of %s free space on disk "
"'%s'. Please free at least an additional %s of disk space on '%s'. Empty "
"your trash and remove temporary packages of former installations using 'sudo "
"apt-get clean'."
msgstr ""
"S'ha interromput l'actualització. Cal un total de %s d'espai lliure al disc "
"«%s», per la qual cosa hauríeu d'alliberar un mínim de %s d'espai addicional "
"al disc «%s». Buideu la paperera i suprimiu els paquets temporals "
"d'anteriors instal·lacions utilitzant l'ordre «sudo apt-get clean»."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1078
#: ../DistUpgrade/DistUpgradeController.py:1899
msgid "Calculating the changes"
msgstr "S'estan calculant els canvis"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1115
msgid "Do you want to start the upgrade?"
msgstr "Voleu iniciar l'actualització?"

#: ../DistUpgrade/DistUpgradeController.py:1188
msgid "Upgrade canceled"
msgstr "S'ha cancel·lat l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1189
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Ara es cancel·larà l'actualització i es restaurarà a l'estat original del "
"sistema. Podeu continuar l'actualització més tard."

#: ../DistUpgrade/DistUpgradeController.py:1195
#: ../DistUpgrade/DistUpgradeController.py:1324
msgid "Could not download the upgrades"
msgstr "No s'han pogut baixar les actualitzacions"

#: ../DistUpgrade/DistUpgradeController.py:1196
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"S'ha interromput l'actualització. Comproveu la connexió a Internet o els "
"suports d'instal·lació i torneu-ho a provar. S'han mantingut tots els "
"fitxers baixats fins ara."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1273
#: ../DistUpgrade/DistUpgradeController.py:1312
#: ../DistUpgrade/DistUpgradeController.py:1417
msgid "Error during commit"
msgstr "S'ha produït un error durant l'enviament"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1275
#: ../DistUpgrade/DistUpgradeController.py:1314
#: ../DistUpgrade/DistUpgradeController.py:1456
msgid "Restoring original system state"
msgstr "S'està restaurant l'estat original del sistema"

#: ../DistUpgrade/DistUpgradeController.py:1276
#: ../DistUpgrade/DistUpgradeController.py:1291
#: ../DistUpgrade/DistUpgradeController.py:1315
msgid "Could not install the upgrades"
msgstr "No s'han pogut instal·lar les actualitzacions"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1281
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"S'ha interromput l'actualització. Pot ser que el vostre sistema hagi quedat "
"en un estat inestable. Ara s'executarà un procés de recuperació (dpkg --"
"configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1286
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug "
"and attach the files in /var/log/dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""
"\n"
"\n"
"Informeu d'aquest error en un navegador a "
"http://bugs.launchpad.net/ubuntu/+source/ubuntu-release-upgrader/+filebug i "
"adjunteu els fitxers de /var/log/dist-upgrade/ a l'informe d'error.\n"
"%s"

#: ../DistUpgrade/DistUpgradeController.py:1325
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"S'ha interromput l'actualització. Verifiqueu la vostra connexió a Internet o "
"el suport d'instal·lació i torneu-ho a intentar. "

#: ../DistUpgrade/DistUpgradeController.py:1405
msgid "Remove obsolete packages?"
msgstr "Voleu suprimir els paquets obsolets?"

#: ../DistUpgrade/DistUpgradeController.py:1406
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Manté"

#: ../DistUpgrade/DistUpgradeController.py:1406
msgid "_Remove"
msgstr "Sup_rimeix"

#: ../DistUpgrade/DistUpgradeController.py:1418
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"S'ha produït un error durant el procés de neteja. Vegeu el següent missatge "
"per a més informació. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1494
msgid "Required depends is not installed"
msgstr "Les dependències requerides no estan instal·lades"

#: ../DistUpgrade/DistUpgradeController.py:1495
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "La dependència requerida «%s» no està instal·lada. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1762
#: ../DistUpgrade/DistUpgradeController.py:1849
msgid "Checking package manager"
msgstr "S'està comprovant el gestor de paquets"

#: ../DistUpgrade/DistUpgradeController.py:1768
#: ../DistUpgrade/DistUpgradeController.py:1774
msgid "Preparing the upgrade failed"
msgstr "S'ha produït un error en la preparació de l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1769
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""
"Ha fallat la preparació del sistema per l'actualització, per la qual cosa "
"s'iniciarà el procés d'informe d'errors."

#: ../DistUpgrade/DistUpgradeController.py:1775
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""
"Ha fallat la preparació de l'actualització del sistema. Per informar d'un "
"error, instal·leu l'apport i executeu «apport-bug ubuntu-release-upgrader»."

#: ../DistUpgrade/DistUpgradeController.py:1792
#: ../DistUpgrade/DistUpgradeController.py:1802
msgid "Getting upgrade prerequisites failed"
msgstr "S'ha produït un error en obtenir els prerequisits de l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1793
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""
"El sistema no ha pogut obtenir els requeriments per a l'actualització. "
"L'actualització s'interromprà i es retornarà el sistema al seu estat "
"original.\n"
"\n"
"També s'iniciarà el procés d'informe d'errors."

#: ../DistUpgrade/DistUpgradeController.py:1803
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""
"El sistema no ha pogut  descarregar els prerequisits per a l'actualització. "
"El procés d'actualització s'aturarà i es retornarà el sistema a l'estat "
"original.\n"
"\n"
"Per informar d'un error instal·leu apport i executeu 'apport-bug ubuntu-"
"release-upgrader'."

#: ../DistUpgrade/DistUpgradeController.py:1833
msgid "Updating repository information"
msgstr "S'està actualitzant la informació dels dipòsits"

#: ../DistUpgrade/DistUpgradeController.py:1840
msgid "Failed to add the cdrom"
msgstr "No s'ha pogut afegir el CD-ROM"

#: ../DistUpgrade/DistUpgradeController.py:1841
msgid "Sorry, adding the cdrom was not successful."
msgstr "No s'ha pogut afegir el CD-ROM correctament."

#: ../DistUpgrade/DistUpgradeController.py:1876
msgid "Invalid package information"
msgstr "La informació del paquet no és vàlida"

#: ../DistUpgrade/DistUpgradeController.py:1877
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""
"Després d'actualitzar la informació dels paquets, no s'ha pogut localitzar "
"el paquet essencial «%s». Això deu al fet que no teniu cap rèplica oficial "
"al llistat de fonts de programari o perquè hi ha una càrrega excessiva en la "
"rèplica que esteu utilitzant. Vegeu /etc/apt/sources.list per veure la "
"llista actual de fonts de programari.\n"
"En cas de rèplica amb sobrecàrrega, podeu intentar l'actualització més tard."

#: ../DistUpgrade/DistUpgradeController.py:1905
#: ../DistUpgrade/DistUpgradeController.py:1963
msgid "Fetching"
msgstr "S'està recollint"

#: ../DistUpgrade/DistUpgradeController.py:1912
#: ../DistUpgrade/DistUpgradeController.py:1967
msgid "Upgrading"
msgstr "S'està actualitzant"

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:1917
#: ../DistUpgrade/DistUpgradeController.py:1969
#: ../DistUpgrade/DistUpgradeController.py:1976
#: ../DistUpgrade/DistUpgradeController.py:1987
msgid "Upgrade complete"
msgstr "S'ha completat l'actualització"

#: ../DistUpgrade/DistUpgradeController.py:1918
#: ../DistUpgrade/DistUpgradeController.py:1970
#: ../DistUpgrade/DistUpgradeController.py:1977
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr ""
"S'ha completat l'actualització, però s'han produït errors durant aquest "
"procés."

#: ../DistUpgrade/DistUpgradeController.py:1926
msgid "Searching for obsolete software"
msgstr "S'està cercant programari obsolet"

#: ../DistUpgrade/DistUpgradeController.py:1939
msgid "System upgrade is complete."
msgstr "S'ha completat l'actualització del sistema."

#: ../DistUpgrade/DistUpgradeController.py:1988
msgid "The partial upgrade was completed."
msgstr "L'actualització parcial s'ha completat."

#: ../DistUpgrade/DistUpgradeFetcher.py:105
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "No s'han trobat les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcher.py:106
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "El servidor potser està sobrecarregat. "

#: ../DistUpgrade/DistUpgradeFetcher.py:118
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "No s'han pogut baixar les notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcher.py:119
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Comproveu la vostra connexió a Internet."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr "autenticació del fitxer «%(file)s» amb la signatura «%(signature)s» "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "s'està extraient «%s»"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "No es pot executar l'eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""
"El més probable és que sigui un error en l'eina d'actualització. Informeu "
"d'aquest error utilitzant l'ordre «ubuntu-bug ubuntu-release-upgrader-core» "
"en un terminal."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "Signatura de l'eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "Eina d'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "Ha fallat la baixada"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Ha fallat l'obtenció de l'actualització. Pot ser que hi hagi algun problema "
"a la xarxa. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "Ha fallat l'autenticació"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Ha fallat l'autenticació de l'actualització. Pot ser que hi hagi algun "
"problema a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "Ha fallat l'extracció"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallat l'extracció de l'actualització. Pot ser que hi hagi algun problema "
"a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "Ha fallat la verificació"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Ha fallat la verificació de l'actualització. Pot ser que hi hagi algun "
"problema a la xarxa o al servidor. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "No es pot executar l'actualització"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Normalment això està provocat per un sistema on /tmp s'ha muntat com a no "
"executable. Torneu-lo a muntar sense l'opció de no executable i torneu a "
"executar l'actualització."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "El missatge d'error és «%s»."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Actualitza"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
msgid "&Cancel"
msgstr "&Cancel·la"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Notes de la versió"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Actualització"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "S'estan baixant els fitxers de paquet addicionals..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Fitxer %s de %s a %s B/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Fitxer %s de %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:124
#: ../DistUpgrade/DistUpgradeViewKDE.py:299
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Inseriu «%s» a la unitat «%s»"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:300
msgid "Media Change"
msgstr "Canvi de suport"

#: ../DistUpgrade/DistUpgradeQuirks.py:183
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""
"Pot ser que el vostre maquinari de gràfics no funcioni bé amb l'Ubuntu 14.04."

#: ../DistUpgrade/DistUpgradeQuirks.py:185
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D Do you still "
"want to continue with the upgrade?"
msgstr ""
"L'entorn d'escriptori «Unity» no és totalment compatible amb el vostre "
"maquinari gràfic. Pot ser que l'escriptori s'executi molt lentament després "
"de l'actualització. El nostre consell és que de moment mantingueu la versió "
"LTS. Per obtenir més informació vegeu "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForUnity3D. Tot i això, "
"voleu continuar amb l'actualització?"

#: ../DistUpgrade/DistUpgradeQuirks.py:209
msgid ""
"Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""
"Sembla que el vostre maquinari gràfic no és totalment compatible amb "
"l'Ubuntu 12.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:211
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""
"La compatibilitat a l'Ubuntu 12.04 LTS per al vostre maquinari gràfic Intel "
"és limitat i podríeu tenir problemes després d'actualitzar. Per obtenir més "
"informació visiteu "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx. Voleu continuar "
"amb l'actualització?"

#: ../DistUpgrade/DistUpgradeQuirks.py:231
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"L'actualització pot reduir els efectes d'escriptori i el rendiment en jocs i "
"altres programes que facin un ús exhaustiu de processament gràfic."

#: ../DistUpgrade/DistUpgradeQuirks.py:235
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Aquest ordinador utilitza el controlador gràfic «fglrx» d'AMD. No hi ha cap "
"versió disponible d'aquest controlador que funcioni amb el vostre maquinari "
"a l'Ubuntu 10.04 LTS.\n"
"\n"
"Voleu continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:272
msgid "No i686 CPU"
msgstr "Sense CPU i686"

#: ../DistUpgrade/DistUpgradeQuirks.py:273
msgid ""
"Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
"extension. All packages were built with optimizations requiring i686 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"El vostre sistema utilitza una CPU i586 o una CPU que no té l'extensió "
"«cmov». Tots els paquets s'han construït amb optimitzacions que requereixen "
"un i686 com a arquitectura mínima. No és possible actualitzar el sistema a "
"una versió nova de l'Ubuntu amb aquest maquinari."

#: ../DistUpgrade/DistUpgradeQuirks.py:310
msgid "No ARMv6 CPU"
msgstr "No és una CPU ARMv6"

#: ../DistUpgrade/DistUpgradeQuirks.py:311
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"El vostre sistema utilitza una CPU ARM que és més antiga que l'arquitectura "
"ARMv6. Tots els paquets del Karmic s'han creat amb optimitzacions que "
"requereixen l'ARMv6 com a arquitectura mínima. No és possible actualitzar el "
"vostre sistema a una versió nova de l'Ubuntu amb aquest maquinari."

#: ../DistUpgrade/DistUpgradeQuirks.py:331
msgid "No init available"
msgstr "No hi ha cap sistema d'inicialització disponible"

#: ../DistUpgrade/DistUpgradeQuirks.py:332
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Segons sembla, el vostre sistema és un entorn virtual sense un dimoni "
"d'inicialització «init», com ara un servidor virtual de Linux (Linux-"
"VServer). L'Ubuntu 10.04 no pot funcionar en aquest tipus d'entorn - abans "
"cal fer una actualització de la configuració de la màquina virtual.\n"
"\n"
"Esteu segur que voleu continuar?"

#: ../DistUpgrade/DistUpgradeQuirks.py:485
msgid "PAE not enabled"
msgstr "El PAE no està habilitat"

#: ../DistUpgrade/DistUpgradeQuirks.py:486
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""
"El sistema incorpora una CPU que no té el PAE habilitat. L'Ubuntu funciona "
"amb sistemes sense PAE només fins a la versió 12.04. Per actualitzar a una "
"versió posterior de l'Ubuntu, haureu d'habilitar el PAE (si és possible). "
"Consulteu la documentació següent per obtenir més informació:\n"
"http://help.ubuntu.com/community/EnablingPAE"

#: ../DistUpgrade/DistUpgradeMain.py:59
msgid "Sandbox upgrade using aufs"
msgstr "Actualització en entorn de proves utilitzant aufs"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Utilitza el camí especificat per cercar un CDROM amb paquets actualitzables"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Utilitzeu un frontal. Actualment es disposa de: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*OBSOLET* aquesta opció s'ignorarà"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Realitza només una actualització parcial (sense reescriure el sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "Inhabilita la compatibilitat amb el GNU screen"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Estableix el «datadir» (directori de dades)"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "Actualitza a la versió de desenvolupament"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:145
#: ../DistUpgrade/DistUpgradeViewKDE.py:313
msgid "Fetching is complete"
msgstr "S'ha completat el recull"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:160
#: ../DistUpgrade/DistUpgradeViewKDE.py:329
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "S'està obtenint el fitxer %li de %li a %s B/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:162
#: ../DistUpgrade/DistUpgradeViewGtk3.py:320
#: ../DistUpgrade/DistUpgradeViewKDE.py:330
#: ../DistUpgrade/DistUpgradeViewKDE.py:480
#, python-format
msgid "About %s remaining"
msgstr "Queden %s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:165
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li"
msgstr "S'està obtenint el fitxer %li de %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:197
#: ../DistUpgrade/DistUpgradeViewKDE.py:370
msgid "Applying changes"
msgstr "S'estan aplicant els canvis"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:223
#: ../DistUpgrade/DistUpgradeViewKDE.py:383
msgid "dependency problems - leaving unconfigured"
msgstr "problemes de dependències - es deixarà sense configurar"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:228
#: ../DistUpgrade/DistUpgradeViewKDE.py:385
#, python-format
msgid "Could not install '%s'"
msgstr "No s'ha pogut instal·lar «%s»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"L'actualització continuarà, però pot ser que el paquet «%s» no sigui "
"funcional. Hauríeu de considerar l'enviament d'un informe d'error sobre "
"aquest fet."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:246
#: ../DistUpgrade/DistUpgradeViewKDE.py:408
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Voleu reemplaçar el fitxer de\n"
"configuració personalitzat «%s»?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:409
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Perdreu tots els canvis realitzats en el fitxer de configuració si trieu "
"reemplaçar-lo per una versió més nova."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:267
#: ../DistUpgrade/DistUpgradeViewKDE.py:432
msgid "The 'diff' command was not found"
msgstr "No s'ha trobat l'ordre «diff»"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:524
#: ../DistUpgrade/DistUpgradeViewText.py:138
msgid "A fatal error occurred"
msgstr "S'ha produït un error greu"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:525
msgid ""
"Please report this as a bug (if you haven't already) and include the files "
"/var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informeu d'això com un error (si no és que ja ho heu fet) i adjunteu els "
"fitxers /var/log/dist-upgrade/main.log i /var/log/dist-upgrade/apt.log al "
"vostre informe. S'ha interromput l'actualització.\n"
"S'ha desat el vostre fitxer sources.list original a "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:555
msgid "Ctrl-c pressed"
msgstr "S'ha premut Ctrl+C"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:556
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Això interromprà l'operació, amb la qual cosa pot ser que es malmeti el "
"sistema. Esteu segur que ho voleu fer?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:694
msgid "To prevent data loss close all open applications and documents."
msgstr ""
"Per a evitar una possible pèrdua de dades, tanqueu tots els documents i "
"aplicacions."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:708
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ja no mantinguts per Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:709
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Es desactualitzaran (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:710
#, python-format
msgid "Remove (%s)"
msgstr "Se suprimiran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:711
#, python-format
msgid "No longer needed (%s)"
msgstr "Ja no són necessaris (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:712
#, python-format
msgid "Install (%s)"
msgstr "S'instal·laran (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:713
#, python-format
msgid "Upgrade (%s)"
msgstr "S'actualitzaran (%s)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:444
msgid "Show Difference >>>"
msgstr "Mostra les diferències >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "<<< Hide Difference"
msgstr "<<< Oculta les diferències"

#: ../DistUpgrade/DistUpgradeViewKDE.py:661
msgid "Error"
msgstr "S'ha produït un error"

#: ../DistUpgrade/DistUpgradeViewKDE.py:899
msgid "&Close"
msgstr "&Tanca"

#: ../DistUpgrade/DistUpgradeViewKDE.py:713
msgid "Show Terminal >>>"
msgstr "Mostra el terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:716
msgid "<<< Hide Terminal"
msgstr "<<< Oculta el terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:788
msgid "Information"
msgstr "Informació"

#: ../DistUpgrade/DistUpgradeViewKDE.py:836
msgid "&Start Upgrade"
msgstr "&Inicia l'actualització"

#: ../DistUpgrade/DistUpgradeViewKDE.py:838
#: ../DistUpgrade/DistUpgradeViewKDE.py:882
#: ../DistUpgrade/DistUpgradeViewKDE.py:885
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detalls"

#: ../DistUpgrade/DistUpgradeViewKDE.py:860
#, python-format
msgid "No longer supported %s"
msgstr "Ja no es mantenen %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:862
#, python-format
msgid "Remove %s"
msgstr "Suprimeix %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:864
#: ../DistUpgrade/DistUpgradeViewText.py:236
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Suprimeix %s (s'havia instal·lat automàticament)"

#: ../DistUpgrade/DistUpgradeViewKDE.py:866
#, python-format
msgid "Install %s"
msgstr "Instal·la %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:868
#, python-format
msgid "Upgrade %s"
msgstr "Actualitza %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:895
#: ../DistUpgrade/DistUpgradeViewText.py:284
msgid "Restart required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeViewKDE.py:895
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr ""
"<b><big>Reinicieu el sistema per a completar l'actualització</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:898
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Reinicia"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:916
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Voleu cancel·lar l'actualització en curs?</big></b>\n"
"\n"
"El sistema pot quedar inusable si la cancel·leu. És molt recomanable "
"continuar amb l'actualització."

#: ../DistUpgrade/DistUpgradeViewKDE.py:920
msgid "Cancel Upgrade?"
msgstr "Voleu cancel·lar l'actualització?"

#: ../DistUpgrade/DistUpgradeView.py:80
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dia"
msgstr[1] "%li dies"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li hora"
msgstr[1] "%li hores"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minut"
msgstr[1] "%li minuts"

#: ../DistUpgrade/DistUpgradeView.py:85
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li segon"
msgstr[1] "%li segons"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:101
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#. 
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#. 
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:119
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:170
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"La baixada trigarà aproximadament %s amb una connexió DSL d'1 Mbit, i "
"aproximadament %s amb un mòdem de 56 k."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:174
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "La baixada trigarà aproximadament %s amb la vostra connexió. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:265 ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "Preparing to upgrade"
msgstr "S'està preparant l'actualització"

#: ../DistUpgrade/DistUpgradeView.py:266
msgid "Getting new software channels"
msgstr "S'estan obtenint canals de programari nous"

#: ../DistUpgrade/DistUpgradeView.py:267 ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Getting new packages"
msgstr "S'estan obtenint els paquets nous"

#: ../DistUpgrade/DistUpgradeView.py:268 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Installing the upgrades"
msgstr "S'estan instal·lant les actualitzacions"

#: ../DistUpgrade/DistUpgradeView.py:269 ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Cleaning up"
msgstr "S'està netejant"

#: ../DistUpgrade/DistUpgradeView.py:357
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d paquet instal·lat ja no està mantingut per Canonical. Així i tot, "
"encara podeu obtenir assistència de la comunitat."
msgstr[1] ""
"%(amount)d dels paquets instal·lats ja no estan mantinguts per Canonical. "
"Així i tot, encara podeu obtenir assistència de la comunitat."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:366
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "Se suprimirà %d paquet."
msgstr[1] "Se suprimiran %d paquets."

#: ../DistUpgrade/DistUpgradeView.py:371
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "S'instal·larà %d paquet nou."
msgstr[1] "S'instal·laran %d paquets nous."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "S'actualitzarà %d paquet"
msgstr[1] "S'actualitzaran %d paquets"

#: ../DistUpgrade/DistUpgradeView.py:385
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Heu de baixar un total de %s. "

#: ../DistUpgrade/DistUpgradeView.py:390
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"La instal·lació de l'actualització pot trigar unes quantes hores. Un cop "
"hagi finalitzat la baixada dels fitxers, el procés no es pot cancel·lar."

#: ../DistUpgrade/DistUpgradeView.py:394
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"La baixada i instal·lació de l'actualització pot trigar unes quantes hores. "
"Un cop hagi finalitzat la baixada dels fitxers, el procés no es pot "
"cancel·lar."

#: ../DistUpgrade/DistUpgradeView.py:399
msgid "Removing the packages can take several hours. "
msgstr "La supressió de paquets pot trigar unes quantes hores. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:404
msgid "The software on this computer is up to date."
msgstr "El programari d'aquest ordinador està actualitzat."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"No hi ha actualitzacions disponibles per al vostre sistema. Ara es "
"cancel·larà l'actualització."

#: ../DistUpgrade/DistUpgradeView.py:418
msgid "Reboot required"
msgstr "Cal que reinicieu el sistema"

#: ../DistUpgrade/DistUpgradeView.py:419
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"L'actualització ha finalitzat i cal reiniciar el sistema. Voleu fer-ho ara?"

#: ../DistUpgrade/DistUpgradeViewText.py:81
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Progrés: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:139
msgid ""
"Please report this as a bug and include the files /var/log/dist-"
"upgrade/main.log and /var/log/dist-upgrade/apt.log in your report. The "
"upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Informeu d'això com un error i adjunteu els fitxers /var/log/dist-"
"upgrade/main.log i /var/log/dist-upgrade/apt.log al vostre informe. S'ha "
"interromput l'actualització.\n"
"S'ha desat el vostre fitxer sources.list original a "
"/etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:163
msgid "Aborting"
msgstr "S'està interrompent"

#: ../DistUpgrade/DistUpgradeViewText.py:168
msgid "Demoted:\n"
msgstr "Degradat:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:175
msgid "To continue please press [ENTER]"
msgstr "Premeu la tecla de retorn per continuar"

#: ../DistUpgrade/DistUpgradeViewText.py:212
#: ../DistUpgrade/DistUpgradeViewText.py:250
#: ../DistUpgrade/DistUpgradeViewText.py:257
msgid "Continue [yN] "
msgstr "Continua [sN] "

#: ../DistUpgrade/DistUpgradeViewText.py:212
#: ../DistUpgrade/DistUpgradeViewText.py:250
msgid "Details [d]"
msgstr "Detalls [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:216
#: ../DistUpgrade/DistUpgradeViewText.py:260
msgid "y"
msgstr "s"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:267
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:222
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:227
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ja no es mantenen: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:232
#, python-format
msgid "Remove: %s\n"
msgstr "Suprimeix: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:242
#, python-format
msgid "Install: %s\n"
msgstr "Instal·la: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:247
#, python-format
msgid "Upgrade: %s\n"
msgstr "Actualitza: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "Continue [Yn] "
msgstr "Continua [Sn] "

#: ../DistUpgrade/DistUpgradeViewText.py:285
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"Cal reiniciar el sistema per a finalitzar l'actualització.\n"
"Si seleccioneu «s», es reiniciarà el sistema."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "S'està baixant el fitxer %(current)li de %(total)li a %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "S'està baixant el fitxer %(current)li de %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Mostra el progrés dels fitxers individuals"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Cancel·la l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Reprèn l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Voleu cancel·lar l'actualització?</big></b>\n"
"\n"
"Si ho feu, pot ser que el sistema sigui inutilitzable. És extremament "
"recomanable continuar amb l'actualització."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Comença l'actualització"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Reemplaça"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Diferències entre els fitxers"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Informa de l'error"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Continua"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Voleu iniciar l'actualització?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr ""
"<b><big>Reinicieu el sistema per a completar l'actualització</big></b>\n"
"\n"
"Deseu la vostra feina abans de continuar."

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Actualització de la distribució"

#: ../data/gtkbuilder/DistUpgrade.ui.h:19
msgid "<b><big>Upgrading Ubuntu to version 15.10</big></b>"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Setting new software channels"
msgstr "S'estan configurant els canals de programari nous"

#: ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Restarting the computer"
msgstr "S'està reiniciant l'ordinador"

#: ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Actualitza"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid ""
"<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr ""
"<b>Hi ha disponible una versió nova de l'Ubuntu. Voleu actualitzar el vostre "
"sistema?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "No actualitzis"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Demana-m'ho més tard"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Actualitza"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Heu triat no actualitzar a la versió nova de l'Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Podeu actualitzar més tard obrint el gestor d'actualitzacions i fent clic a "
"«Actualitza»."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Realitza una actualització de la versió"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Cal que us autentiqueu per actualitzar l'Ubuntu."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr "Realitza una actualització parcial"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr "Cal que us autentiqueu per fer una actualització parcial."

#: ../do-partial-upgrade:75 ../do-release-upgrade:79
msgid "Show version and exit"
msgstr "Mostra la versió i surt"

#: ../do-partial-upgrade:78 ../do-release-upgrade:86
msgid "Directory that contains the data files"
msgstr "El directori que conté els fitxers de dades"

#: ../do-partial-upgrade:81 ../do-release-upgrade:99
msgid "Run the specified frontend"
msgstr "Executa el frontal especificat"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "S'està executant una actualització parcial"

#: ../do-release-upgrade:33 ../do-release-upgrade:45
msgid "Downloading the release upgrade tool"
msgstr "S'està baixant l'eina d'actualització a una versió nova"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr ""
"Comprova si és possible actualitzar a la darrera versió de desenvolupament"

#: ../do-release-upgrade:89 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Proveu d'actualitzar a l'última versió fent servir l'actualitzador de "
"$distro-proposed"

#: ../do-release-upgrade:93
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Executeu un mode d'actualització especial.\n"
"Actualment disposeu del mode «desktop» per a actualitzacions de sistemes "
"d'escriptori i del mode «server» per a sistemes servidors."

#: ../do-release-upgrade:100
msgid "Test upgrade with a sandbox aufs overlay"
msgstr "Prova d'actualitzar en un entorn de proves amb aufs"

#: ../do-release-upgrade:102
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Només verifica si hi ha disponible una versió nova i informa del resultat "
"mitjançant el codi de sortida"

#: ../do-release-upgrade:116
msgid "The options --devel-release and --proposed are"
msgstr "Les opcions --devel-release i --proposed són"

#: ../do-release-upgrade:117
msgid "mutually exclusive. Please use only one of them."
msgstr "mútuament excloents. Feu servir només una d'elles."

#: ../do-release-upgrade:121
msgid "Checking for a new Ubuntu release"
msgstr "S'està cercant una versió nova de l'Ubuntu"

#: ../do-release-upgrade:133
msgid "Your Ubuntu release is not supported anymore."
msgstr "La vostra versió de l'Ubuntu ja no està mantinguda."

#: ../do-release-upgrade:134
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"Per obtenir més informació sobre l'actualització, aneu a:\n"
"%(url)s\n"

#: ../do-release-upgrade:132
msgid "No new release found"
msgstr "No s'ha trobat cap versió nova"

#: ../do-release-upgrade:149
msgid "Release upgrade not possible right now"
msgstr "En aquests moments no es pot efectuar l'actualització de versió"

#: ../do-release-upgrade:150
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"En aquests moments no es pot efectuar l'actualització de versió. Torneu-ho a "
"provar més tard. El servidor ha respost: «%s»"

#: ../do-release-upgrade:156
#, c-format
msgid "New release '%s' available."
msgstr "La versió nova «%s» ja està disponible."

#: ../do-release-upgrade:157
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Executeu «do-release-upgrade» per actualitzar a la nova versió."

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr "Gestor d'actualitzacions"

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Hi ha disponible una actualització de l'Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Heu triat no actualitzar a l'Ubuntu %s"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr "Afegeix informació de depuració"

#, python-format
#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "S'ha produït un problema irresoluble mentre es calculava l'actualització:\n"
#~ "%s\n"
#~ "\n"
#~ "Això pot ser degut a:\n"
#~ " * l'actualització a una versió en desenvolupament de l'Ubuntu\n"
#~ " * l'execució de l'actual versió en desenvolupament de l'Ubuntu\n"
#~ " * la instal·lació de paquets no oficials de programari i no proporcionats "
#~ "per l'Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect which "
#~ "version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "No teniu instal·lats el paquet ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop o edubuntu-desktop i no s'ha pogut detectar la versió de l'Ubuntu "
#~ "que esteu utilitzant.\n"
#~ "Instal·leu un d'aquests paquets abans de continuar; per fer-ho podeu "
#~ "utilitzar el Synaptic o l'apt-get"

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "El paquet «debsig-verify» està instal·lat"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first and "
#~ "run the upgrade again."
#~ msgstr ""
#~ "L'actualització no pot continuar amb aquest paquet instal·lat.\n"
#~ "Elimineu-lo amb el Synaptic o amb l'ordre «apt-get remove debsig-verify» i "
#~ "torneu a executar l'actualització."

#, python-format
#~ msgid ""
#~ "While scanning your repository information no mirror entry for the upgrade "
#~ "was found. This can happen if you run a internal mirror or if the mirror "
#~ "information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
#~ "here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "No s'ha trobat cap entrada de rèplica per a l'actualització a la vostra "
#~ "informació sobre els dipòsits. Això pot ésser degut a què estigueu executant "
#~ "una rèplica interna o que les dades de les rèpliques no estiguin "
#~ "actualitzades.\n"
#~ "\n"
#~ "Voleu sobreescriure el fitxer «sources.list» de totes maneres? Si trieu que "
#~ "«Sí», totes les entrades «%s» s'actualitzaran a «%s». Si trieu que «No», es "
#~ "cancel·larà l'actualització."

#~ msgid "evms in use"
#~ msgstr "S'està utilitzant l'evms"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "El vostre sistema utilitza el gestor de volum «evms» a /proc/mounts. Aquest "
#~ "programari no es mantindrà més, desconnecteu-lo i torneu a executar "
#~ "l'actualització."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Aquest ordinador utilitza el controlador gràfic «nvidia» d'NVIDIA. No hi ha "
#~ "cap versió disponible d'aquest controlador que funcioni amb el vostre "
#~ "maquinari a l'Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Voleu continuar?"

#~ msgid "Authentication is required to perform a release upgrade"
#~ msgstr "Cal autenticació per realitzar una actualització de la versió"

#~ msgid "Authentication is required to perform a partial upgrade"
#~ msgstr "Cal autenticació per realitzar una actualització parcial"

#~ msgid "<b><big>Upgrading Ubuntu to version 15.04</big></b>"
#~ msgstr "<b><big>Actualitzant Ubuntu a la versió 15.04</big></b>"
